src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -29,12 +29,15 @@
     <a href="/conversations">Conversations</a>
     <% if(request.getSession().getAttribute("user") != null){ %>
       <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+      <a  href="/userprofile/<%= request.getSession().getAttribute("user")%>">About <%= request.getSession().getAttribute("user")%></a>
```

Comment:
I might advocate for just making the previous "Hello <user>" clickable.  Or removing that and just leaving "About <user>"